README.md

```diff
@@ -24,3 +24,5 @@ To trigger a release of Mattermost Push-Proxy, follow these steps:
     make major
     ```
    This will release a major change.
+
+See the [CHANGELOG](CHANGELOG.md) for more information.
```

Comment:
No need for this line. It doesn't look like the CHANGELOG is maintained anyways.